Make watchdogReset() force_inline

DRracer opened the pull request:
It makes no sense keeping watchdogReset as a separate function which must be called, since it only contains one instruction: `wdr`.

Not only was the code larger by 32 bytes in total, but also much slower (call+ret take 4 cycles together for no reason in this case).

Surprisingly, doing just this on FW 3.9.3 solves issue #2954 on the one affected EINSY board, even though it makes not much sense (there must be some other timing issue on that one board itself).

Commit:
Make watchdogReset() force_inline

It makes no sense keeping watchdogReset as a separate function which
must be called, since it only contains one instruction: "wdr".

Not only was the code larger by 32 bytes in total, but also much slower
(call+ret take 4 cycles together for no reason in this case).

Surprisingly, doing just this on FW 3.9.3 solves issue #2954 on the one
affected EINSY board, even though it makes not much sense (there must be
some other timing issue).

leptun reviewed (approved):
For the near future: replace this reimplementation of WDT control with the avr-libc one (`<avr/wdt.h>`)

Commit:
Use standard wdt_reset() from AVR lib

which translates to the one `wdr` instruction like before
+ wrap configuration of watchdog into cli/sei
